networkdb/cluster.go

```diff
@@ -141,6 +142,7 @@ func (nDB *NetworkDB) clusterInit() error {
 		{config.GossipInterval, nDB.gossip},
 		{config.PushPullInterval, nDB.bulkSyncTables},
 		{retryInterval, nDB.reconnectNode},
+		{deadNodeReapInterval, nDB.reapDeadNode},
```

Comment:
@mrjana Relying on wall clock time can lead to failures if the time changes. That is why I opted for this approach because in this case removing the dead nodes is not strictly time bound.